app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,24 @@ def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
+
+  def new_comment
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:id])
+  end
```

Comment:
Added comments controller and shifted methods to there

Reply:
- [ ] Instead of creating the sample comments inside the db/seed, kindly add the form for the comments so that any user can comment depending on the post.

Reply:
![image](https://user-images.githubusercontent.com/55635977/157853270-bd9cdf3c-ceaf-4c7a-8a02-348b5d1f5982.png)

Reply:
![image](https://user-images.githubusercontent.com/55635977/157853321-94788aa8-ca59-4db6-aaab-283e6ae5820c.png)

Reply:
I did that as well, where the comments are displayed there are 2 links, one says add like and one says add comment. The comments in seed are just to start it up with some data